kotlin-extensions/src/main/kotlin/kotlinx/js/WeakMap.kt

```diff
@@ -0,0 +1,10 @@
+package kotlinx.js
+
+external class WeakMap<in K : Any, out V>(
```

Comment:
It seems `V` parameter should be invariant because `WeakMap` is mutable

Reply:
Fixed